tachyon/math/finite_fields/binary_fields/binary_fields.h

```diff
@@ -5,9 +5,64 @@
 #ifndef TACHYON_MATH_FINITE_FIELDS_BINARY_FIELDS_BINARY_FIELDS_H_
 #define TACHYON_MATH_FINITE_FIELDS_BINARY_FIELDS_BINARY_FIELDS_H_
 
+// clang-format off
+#include "tachyon/math/finite_fields/binary_fields/binary_field1_config.h"
+#include "tachyon/math/finite_fields/binary_fields/binary_field2_config.h"
+#include "tachyon/math/finite_fields/binary_fields/binary_field4_config.h"
+#include "tachyon/math/finite_fields/binary_fields/binary_field8_config.h"
+#include "tachyon/math/finite_fields/binary_fields/binary_field16_config.h"
+#include "tachyon/math/finite_fields/binary_fields/binary_field32_config.h"
+#include "tachyon/math/finite_fields/binary_fields/binary_field64_config.h"
+// clang-format on
+#include "tachyon/math/finite_fields/binary_field_traits_forward.h"
+
+namespace tachyon::math {
+
+template <>
+struct BinaryFieldTraits<BinaryField1Config> {
+  using SubConfig = BinaryField1Config;
+};
+
+template <>
+struct BinaryFieldTraits<BinaryField2Config> {
+  using SubConfig = BinaryField2Config;
+};
+
+template <>
+struct BinaryFieldTraits<BinaryField4Config> {
+  using SubConfig = BinaryField4Config;
+};
+
+template <>
+struct BinaryFieldTraits<BinaryField8Config> {
+  using SubConfig = BinaryField8Config;
+};
+
+template <>
+struct BinaryFieldTraits<BinaryField16Config> {
+  using SubConfig = BinaryField8Config;
```

Comment:
Why are the following binary field configs the same or different? (aka 16&8 vs 8&8?)
```
struct BinaryFieldTraits<BinaryField16Config> {
  using SubConfig = BinaryField8Config;
}
```
and 
```
struct BinaryFieldTraits<BinaryField8Config> {
  using SubConfig = BinaryField8Config;
};
```

Reply:
This is because when taking a look at `binary_tower_operations.h`, the binary fields whose modulus bits are greater than 8 are implemented in a recursive way that they are decomposed into smaller binary fields so called `SubBinaryField`.